### ISSUE TITLE
provide more useful deb changelog

### ISSUE THREAD

**Issue by jdstrand** [zabbly/incus#24]
Thanks for the zabbly incus packages. With the recent 0.5.x updates, I found myself wanting to know what the changes were to the debs, so I looked at /usr/share/doc/incus/changelog.Debian.gz only to find the single entry:

```
incus (0.5.1-202401301901-ubuntu22.04) jammy; urgency=medium

  * Automated Incus stable build

 -- Zabbly ZFS builds <info@zabbly.com>  Tue, 30 Jan 2024 19:01:41 +0000
```

Ideally this would have git commits and oneline summaries (or similar) going back to at least the previous minor version (but really, why not all of them).

**Comment by stgraber** [comment]
Yeah, I'll have to see what I can go to generate it from the git history as I don't trust myself to keep it up to date by myself :)

**Comment by a-roussos** [comment]
It might be worth having a look at [git-cliff](https://github.com/orhun/git-cliff) for this task.

**Comment by gl-yziquel** [comment]
@stgraber

The reno tool from openstack is also quite cool. Don't know how much it would fit debian changelos specifically, but the logic of how release notes can be kept in git and get generated at release time is definitely something I like in reno. It may help you on the "I don't trust myself to keep it up to date by myself" part of things.
